climpred/classes.py

```diff
@@ -1080,6 +1091,9 @@ def _verify(
                     ref = xr.concat(metric_over_leads, dim='lead', **CONCAT_KWARGS)
                     ref['lead'] = forecast['lead']
                     result = xr.concat([result, ref], dim='skill', **CONCAT_KWARGS)
+            # rename back to 'init'
+            if 'time' in result.dims:
+                result = result.rename({'time': 'init'})
```

Comment:
yes this is permanent. we have this already in `compute_hindcast` but I didnt recognize that function is not called directly anymore.

initialized data has the "time-like" dimension `init`, while verif has dim `time`. we need to rename. so before skill calculation we renamed ininitialized from `init` to `time`, now we go back. renaming verif to init would be easier but is weird because only time sounds reasonable for verif.